docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -46,6 +46,86 @@ apb init
 
 NOTE: Best practice is to name the APB ending with `-apb`.
 
+== Making your APB consumable as a mobile service
+
+When creating an APB for a mobile service, where possible use Kubernetes resources
```

Comment:
I think having an example here would be good to highlight what is meant. e.g. Use a Deployment instead of a DeploymentConfig. Avoid BuildConfig/ImageStream objects if possible